src/codeu/chat/DerbyStore.java

```diff
@@ -35,23 +35,24 @@
 	private final String messageTableName = "message";
 	private final String chatParticipantsTableName = "chatParticipants";
 	
-	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getConversationInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getUsersInvolved = "SELCT * FROM " + chatParticipantsTableName + " WHERE id = ?" ;
+	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE userid = ?";
```

Comment:
Yeah that's why it's okay for our purposes. But in "real life" you'd probably want to be specific about exactly what columns you're getting, even if you're getting them all. Not saying you have to change it here, just something to think about.